src/worker.js

```diff
@@ -212,6 +212,99 @@ const queue = asyncQueue(({ id, data }, taskCallback) => {
 
             return options;
           },
+          getLogger: (name) => {
```

Comment:
Should we also support the situation where `name` is a function like the Compilation.getLogger did?

https://github.com/webpack/webpack/blob/899f06934391baede59da3dcd35b5ef51c675dbe/lib/Compilation.js#L1195-L1202

Reply:
Yeah, we need it

Reply:
8d39667